module-check/src/test/clojure/clojure/core/typed/test/repl_new.clj

```diff
@@ -0,0 +1,29 @@
+(ns clojure.core.typed.test.set_new
+  (:require [clojure.core.typed :as t]
+            [clojure.set :as set] 
+            [clojure.test :refer :all]                
+            [clojure.core.typed.test.test-utils :refer :all]))
+
+(deftest apropos-test
+  (is-tc-e (apropos "clojure") (clojure.lang.LazySeq String)        
+             :requires [[clojure.repl :refer [apropos]]])
+  (is-tc-err (apropos "clojure") t/Str 
+             :requires [[clojure.repl :refer [apropos]]])
+  (is-tc-err (apropos 'clojure) t/Str 
```

Comment:
This case fails both because argument type doesn't match (passed symbol, but function takes String) and return value doesn't match. I think we should rather test only invalid argument  and use correct return type, as for return type you already have check above.